Topics/Tech_Stacks/Django_Rest.md

```diff
@@ -62,6 +67,24 @@ class TodoItem(models.Model):
     completed = models.BooleanField(default=False)
 ```
 
+For this model, we have used a CharField for the string title. The CharField is used for small amounts of text, with a default max length of 255 characters. We have defined the max length to be 200 characters in this case. We have also used a BooleanField for the boolean completion status, and we have explicitly defined the default value to be False. Thus, whenever a value isn't provided for the completed field, it will automatically be set to False. 
```

Comment:
I think the second sentence should start with "A CharField"

Reply:
[ea08aee](https://github.com/learning-software-engineering/learning-software-engineering.github.io/pull/310/commits/ea08aee1dd22045fff731064ae7c678a898aa516)